rabbitmq.info.yml

```diff
@@ -2,5 +2,4 @@ name: "RabbitMQ"
 description: "RabbitMQ Integration for Drupal"
 core: "8.x"
 type: "module"
-php: "5.6"
 package: "Performance and scalability"
```

Comment:
The main changes in 5.6 are the spread operator (... : not used here), and the ability to initialize constants with computed values (need to check).